### ISSUE TITLE
Telemetry in Docker Containers

### ISSUE THREAD

**Issue by jcooper01** [microsoft/navcontainerhelper#1514]
**Describe the bug**
When we add our Applicaiton Insights key to our app.json and issue LOGMESSAGE commands in an app running in a Docker container, nothing appears in our Application Insights reporting. We don't know if this would work in the production SaaS/sandbox environments.

**To Reproduce**
Steps and to reproduce the behavior:
We are using https://bcartifacts.azureedge.net/sandbox/17.0.17126.18816/us and BC-ContainerHelper. When we publish an extension to the container that has our Applicaiton Insights Key in the app.json and execute codeunits that have LOGMESSAGE commands, we do not see those messages in Applicaiton Insights.

``` AL Code to reproduce the issue ```

**Expected behavior**
Applicaiton Insights Log should show telemetry data

**Screenshots**

**5. Versions:**
BC17

**Comment by freddydk** [comment]
I am working on enabling telemetry in docker in general - will check apps as well.
This issue https://github.com/microsoft/navcontainerhelper/issues/1506 is also around telemetry.

**Comment by freddydk** [comment]
I added Telemetry in app.json on my BingMaps sample here: https://dev.azure.com/businesscentralapps/_git/BingMaps and added a LogMessage in OnPageOpen in the Setup dialog.
Checked in and after my pipelines succeeded, I installed the app using: 

```
Publish-BcContainerApp -appFile "https://businesscentralapps.blob.core.windows.net/bingmaps-preview/latest/apps.zip" -skipVerification -sync -install -credential $credential
```
I saw this in the output:
![image](https://user-images.githubusercontent.com/10775043/101235466-2579d580-36c9-11eb-9c42-5cd7ff68a10b.png)

Surprised to see a Telemetry test log message before the installation (actually 10 minutes before) until I realized that the first logmessage of course was during the test execution on my build server.

So, I would claim that it works fine.
Have you checked that your container has internet access and can do DNS lookups?
